vms/platformvm/txs/executor/staker_tx_verification_test.go

```diff
@@ -269,13 +295,16 @@ func TestVerifyAddPermissionlessValidatorTx(t *testing.T) {
 				bootstrapped := &utils.Atomic[bool]{}
 				bootstrapped.Set(true)
 				return &Backend{
-					Ctx:          snow.DefaultContextTest(),
+					Ctx: snow.DefaultContextTest(),
+					Config: &config.Config{
+						DurangoTime: time.Time{}, // activate latest fork
+					},
 					Bootstrapped: bootstrapped,
 				}
 			},
 			stateF: func(ctrl *gomock.Controller) state.Chain {
 				state := state.NewMockChain(ctrl)
-				state.EXPECT().GetTimestamp().Return(time.Unix(0, 0))
+				state.EXPECT().GetTimestamp().Return(time.Unix(1, 0)) // must be after durango fork
```

Comment:
again we should probably define when durango occurs and use that variable here accordingly.